utils/copyImagesToQuay.txt

```diff
@@ -0,0 +1,20 @@
+# list images to copy to quay.io/eclipse/che-*, 1 per line
+# lines starting with # will be ignored 
+# see copyImagesToQuay.sh for skopeo copy script
+
+docker.io/centos/mongodb-36-centos7
+docker.io/centos/mysql-57-centos7
+docker.io/centos/postgresql-96-centos7
+docker.io/dirigiblelabs/dirigible-openshift:3.4.0
+docker.io/ksmster/che-editor-jupyter:5.7.0
```

Comment:
For this image it could be moved by @skabashnyuk

Reply:
meanwhile copied to https://quay.io/repository/eclipse/che--ksmster--che-editor-jupyter

Reply:
I see your thumbs-down and raise you a "the release is broken".  :P

Reply:
this image should just be dropped from your script